bundle/downstream/manifests/quay-bridge-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,513 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    olm.skipRange: ">=3.5.x <3.6.0"
```

Comment:
@bentito is there any difference between `">=3.5.x <3.6.0"` and `">=3.5.0 <3.6.0"` or they mean the same thing?